README.md

```diff
@@ -1598,6 +1598,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [SMak06](https://github.com/SMak06)
   - [smir45](https://github.com/smir45)
   - [Snehal Senapati](https://github.com/snehalsenapati23)
+  - [Simon Shurety](https://github.com/SimonjShurety)
```

Comment:
Please add your name in line 1595, after Silas Omurunga.
Let us know if you need any help with that. 🙂